plugin-api/src/main/java/io/everytrade/server/model/SupportedExchange.java

```diff
@@ -14,34 +14,36 @@
 @AllArgsConstructor
 @FieldDefaults(level = PRIVATE, makeFinal = true)
 public enum SupportedExchange {
+    ANYCOIN("Anycoin", "anycoin"),
+    AQUANOW("Aquanow", "aquanow"),
+    BINANCE("Binance", "binance"),
     BITSTAMP("Bitstamp", "bitstamp"),
     BITTREX("Bittrex", "bittrex"),
-    GENERAL_BYTES("General Bytes CAS", "generalbytes"),
```

Comment:
Please keep the supported exchange as are but just add new one

Reply:
fixed